addons/suggest_locations/README.md

```diff
@@ -0,0 +1,53 @@
+# Suggested Locations
+
+Module to handle suggested locations
+
+# Dependencies
+- base
+- get_info 
+- udes_common (requires groupby)
+- udes_stock (requires get_empty children)
+
+# Policies (u_suggest_location_policy)
+
+Policies are based on a metaclass that enforces the methods needed for suggested locations to work. Essentially they are used to govern how stock is moved within a warehouse, along with the u_drop_location_constaint.
+
+| Policy | Description |
+| - | - |
+| ByProduct | Match drop locations to locations with existing products of the same type are |
+| MatchByMoveline | Suggest the location that is already expected for the move line |
+
+# Drop Configurations (u_drop_location_constaint)
+
+For flexibility there are a range of different configurations on how items can be dropped in a warehouse, ranging from ones where no checks are done giving the user freedom of choice, to ones that are more restrictive and only allow the item to be dropped in a particular set of locations. 
+
+| Name | Description |
+| - | - |
+| do_not_scan | Don't require scanning the destination location; a user has complete autonomy and can place products where they wish with no checks |
```

Comment:
When we don't scan destination location I think it is because the user don't have much of a choice (probably there is only one location) or it is just asked to confirm the destination location (already set in the move line) by pressing button. So I would remove or reword the second part of the description since atm is the same as `scan`.